include/cudaq/Optimizer/Transforms/Passes.td

```diff
@@ -975,6 +975,29 @@ def RegToMem : Pass<"regtomem", "mlir::func::FuncOp"> {
   let dependentDialects = ["cudaq::cc::CCDialect", "quake::QuakeDialect"];
 }
 
+def RemoveUselessStores : Pass<"remove-useless-stores", "mlir::func::FuncOp"> {
+  let summary = "Remove stores that are overriden by other stores.";
+  let description = [{
+
```

Comment:
Can we add more words here? I think the example is good, but examples aren't the best documentation as they can leave some unanswered questions.